README.md

```diff
@@ -110,6 +110,7 @@ Additional lists you might find useful:
 - [TinyAuth plugin](https://github.com/dereuromark/cakephp-tinyauth) - Authentication and role based (single/multi) authorization as very light-weight approach.
 - [Tools:Passwordable](https://github.com/dereuromark/cakephp-tools) - Containing [Passwordable behavior](https://github.com/dereuromark/cakephp-tools/blob/master/docs/Behavior/Passwordable.md) for a DRY approach on password hashing.
 - [TwoFactorAuth plugin](https://github.com/andrej-griniuk/cakephp-two-factor-auth) - Allows two factor authentication using Google Authenticator or similar app to generate one-time codes. Based on [RobThree/TwoFactorAuth](https://github.com/RobThree/TwoFactorAuth) library.
+- [ApiTokenAuthenticator plugin](https://github.com/rrd108/api-token-authenticator) - A Simple Token Authentication Plugin for CakePHP 4 REST API-s.
```

Comment:
Order matters

Please also note:
Commiting lock file in libraries is usually not a thing.
I also dont see any CI running to ensure the tests are OK.

Reply:
```suggestion
- [ApiTokenAuthenticator plugin](https://github.com/rrd108/api-token-authenticator) - A simple token authentication plugin for CakePHP REST APIs.
```

Reply:
> Order matters

No problem. I did not know.

> Please also note: Commiting lock file in libraries is usually not a thing. I also dont see any CI running to ensure the tests are OK.

composer.lock removed
CI added for php 7.4 and 8.1

Reply:
https://github.com/FriendsOfCake/awesome-cakephp/blob/master/CONTRIBUTING.md

Reply:
Since the plugin depends on cajephp/authentication 2.0+, you can bump the cakephp version to ^4.4 instead of ^4.0 and also rename the class `Plugin` to `ApiTokenAuthenticatorPlugin` as per the new convention.

Reply:
required CakePHP version bumped to 4.4
Plugin file renamed for the the convention

Reply:
FYI: You got some noise files and commits in there: https://github.com/rrd108/api-token-authenticator/tree/main/src
And also: You dont have to make new releases for every semantic (non functional) change, e.g. test changes.

Reply:
> FYI: You got some noise files and commits in there: https://github.com/rrd108/api-token-authenticator/tree/main/src 

I removed that noise files. How did you find them? 

> And also: You dont have to make new releases for every semantic (non functional) change, e.g. test changes.

Yep, I know but I changed what php version is supported, that is why I made a new release.

Reply:
I was referring to tests added and CI added :)

Reply:
I dont see any of the requested changes to the PR here yet inside.